.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -25,12 +25,17 @@ If applicable, add screenshots to help explain your problem.
 
 **Please complete the following information:**
 
-- OS: [e.g. iOS]
+- Operating System: [e.g. Windows 11]
+- Processor Architecture: [Arm/x86/Unknown]
```

Comment:
you could add more if you want 
https://en.wikipedia.org/wiki/Comparison_of_instruction_set_architectures

Reply:
The only ones we support are Arm and x86 so I think its fine lol. Even though I've worked with MIPs assembly before, I doubt we will ever support RimSort MIPs.